tests/DependencyInjection/ConfigurationTest.php

```diff
@@ -159,7 +159,7 @@ public function testItThrowsAnErrorOnInvalidModeConfig(): void
     public function testItThrowsAnErrorOnMissingModeConfig(): void
     {
         $this->expectException(InvalidConfigurationException::class);
-        $this->expectExceptionMessage('The child node "mode" at path "docusign.demo" must be configured.');
+        $this->expectExceptionMessage('The child config "mode" under "docusign.demo" must be configured: Type of signature to use: remote or embedded.');
```

Comment:
```suggestion
        $this->expectExceptionMessage('The child config "mode" under "docusign.demo" must be configured. You can use either remote or embedded.');
```

Reply:
This is an automated message from symfony/dependency-injection, I cannot update it easily

Reply:
This will impact the bundle configuration, as it uses the node info here. Still ok about it @GregoireHebert ?

Reply:
it's just a message, and the CI dependency right?
you didn't change anything else 🤔

Reply:
I don't understand your last message...
It's an error message in case of invalid configuration. This error message is built automatically from symfony/dependency-injection, using the "info" node option from our bundle configuration.

Reply:
oh ok,I didn't notice this was used and generated in that way.
thanks Vincent !